sema/bench/handler.py

```diff
@@ -0,0 +1,79 @@
+import os
+from logging import getLogger
+
+from pyshacl import validate
+
+from sema.harvest import service as TravHarv
+from sema.subyt import Subyt
+from sema.syncfs import SyncFsTriples
+
+logger = getLogger(__name__)
+
+# TODO - for subyt and syncfs, import service from sema
+
+
+class TaskHandler:
+    def handle(self):
+        raise NotImplementedError
+
+
+class CSVWHandler(TaskHandler):
+    def handle(self, task):
+        # TODO: implement
+        raise NotImplementedError
+
+
+class EyereasonerHandler(TaskHandler):
+    def handle(self, task):
+        # TODO: implement
+        raise NotImplementedError
+
+
+class Pykg2tblHandler(TaskHandler):
```

Comment:
got renamed to sema.query
still needs a redo of the __main__ and a ServiceBase implementation